app/controllers/api/v2/hostgroups_controller.rb

```diff
@@ -56,6 +56,8 @@ def show
           param :group_parameters_attributes, Array, :required => false, :desc => N_("Array of parameters") do
             param :name, String, :desc => N_("Name of the parameter"), :required => true
             param :value, String, :desc => N_("Parameter value"), :required => true
+            param :parameter_type, Parameter::KEY_TYPES, :desc => N_("Type of value"), :required => true
+            param :hidden_value, [true, false]
```

Comment:
Shouldn't this (and in hosts) be 
```suggestion
            param :hidden_value, :bool
```

Reply:
@tbrisker thanks for review. It was like that in `common_parameters` and I blindly made the same thing in host/hostgroup, but yes totally agree. It's changed.